src/main.ts

```diff
@@ -1,24 +1,128 @@
-import './style.css'
-import typescriptLogo from './typescript.svg'
-import viteLogo from '/vite.svg'
-import { setupCounter } from './counter.ts'
-
-document.querySelector<HTMLDivElement>('#app')!.innerHTML = `
-  <div>
-    <a href="https://vite.dev" target="_blank">
-      <img src="${viteLogo}" class="logo" alt="Vite logo" />
-    </a>
-    <a href="https://www.typescriptlang.org/" target="_blank">
-      <img src="${typescriptLogo}" class="logo vanilla" alt="TypeScript logo" />
-    </a>
-    <h1>Vite + TypeScript</h1>
-    <div class="card">
-      <button id="counter" type="button"></button>
-    </div>
-    <p class="read-the-docs">
-      Click on the Vite and TypeScript logos to learn more
-    </p>
-  </div>
-`
-
-setupCounter(document.querySelector<HTMLButtonElement>('#counter')!)
+import "./style.css";
+import {
+  Config,
+  FRUser,
+  TokenManager,
+  UserManager,
+} from "@forgerock/javascript-sdk";
+
+Config.set({
+  clientId: "aj-public-sdk-client", // e.g. 'ForgeRockSDKClient'
+  redirectUri: `http://localhost:3000`, // e.g. 'https://sdkapp.example.com:8443/callback.html'
+  scope: "openid profile email address revoke", // e.g. 'openid profile email address phone'
+  serverConfig: {
+    baseUrl: "https://openam-sdks.forgeblocks.com/am", // e.g. 'https://myorg.forgeblocks.com/am' or 'https://openam.example.com:8443/openam'
+    timeout: 3000, // 90000 or less
+  },
+  realmPath: "alpha", // e.g. 'alpha' or 'root'
+  // tree: "AJ Choice Login", // e.g. 'sdkAuthenticationTree' or 'Login'
+});
+
+interface LoginOptions {
+  forceRenew?: boolean;
+}
+
+const userElem = document.getElementById("user");
+
+async function handleLogin(
+  _event: Event,
+  options?: LoginOptions
+): Promise<void> {
+  const { forceRenew } = options ?? {};
+  try {
+    // Start authenitcation flow
+    // 'redirect' option enables central login directly with auth server
+    hideUser();
```

Comment:
why do we need to call `hideUser` here?

Reply:
I was thinking that whenever the flow restarts it could be different user so we don't want to keep displaying the last user that logged in. However, I'm now realizing that the page will refresh for a new user anyways and showUser just overrides the user element's contents so this really isn't needed.